gulpfile.js

```diff
@@ -0,0 +1,43 @@
+(function() {
```

Comment:
I'm so used to them that I do one in every js file. By it's not mandatory, I'll remove it.